src/components/NewMessageDetail.vue

```diff
@@ -261,6 +261,7 @@ export default {
 			})
 		},
 		sendMessage(data) {
+			data.body = '<style>p{margin:0;}</style>' + data.body
```

Comment:
Two remarks

1) This is problematic for plaintext messages as at this point we already have a non-hmtl version of the body AFAIK
2) The CKEditor would allow us to change the way certain elements are rendered into HTML. We could change the generated HTML there as well. Might be the cleaner approach :thinking:

Reply:
>     1. This is problematic for plaintext messages as at this point we already have a non-hmtl version of the body AFAIK

You're right. 

>     2. The CKEditor would allow us to change the way certain elements are rendered into HTML. We could change the generated HTML there as well. Might be the cleaner approach 🤔

You mean using this? https://ckeditor.com/docs/ckeditor5/latest/features/headings.html

Reply:
More like this https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/conversion/conversion-extending-output.html#adding-a-css-class-to-block-elements. But the question is how to add the styles. We could either have them inline for every `<p>` or include a style block somewhere.

Reply:
I would say adding a style bock is cleaner. Isn't?

Reply:
wouldn't this be more appropriate? 

https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/conversion/conversion-preserving-custom-content.html

Reply:
Makes sense!